sync/handler_download_request.go

```diff
@@ -52,6 +77,8 @@ func (handler *downloadRequestHandler) ParsPayload(p payload.Payload, initiator
 			break
 		}
 	}
+	// To avoid sending blocks again, we update height for this peer
+	peer.UpdateHeight(from - 1)
```

Comment:
given its decentralized network i don't see how sure we are about applicant getting the response

Reply:
We need to have a mechanism to prevent sending blocks to a node repeatedly. In a decentralize network this make sense, A node can ask other nodes to send the same blocks but not from the one who asked before.

Reply:
@b00f 
sure but as the network is too small to have many archive nodes i think we need to make exceptions